stream-compositions/services/legal-entity-composition-service/src/test/resources/mappings/mapping-service-api-v2-users-identities-GUNJl.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "d1d22565-5b5f-39d8-94c6-d33028dfaddb",
+  "request": {
+    "url": "/service-api/v2/users/identities",
+    "method": "POST",
+    "bodyPatterns": [
+      {
+        "equalToJson": "{\"externalId\":\"gc-party\",\"legalEntityInternalId\":\"2c9280899126d76f01912cbb0a420006\",\"fullName\":\"Sachin Ramesh Tendulkar\",\"emailAddress\":\"gc-party@example.com\",\"mobileNumber\":\"9888887888\",\"givenName\":null,\"familyName\":null,\"attributes\":{},\"additions\":{}}",
```

Comment:
Grand Central does provide family name, why is it not expected to be passed? If seen scary code in project (or modelbank?) code to "split the full name into first and last part". 😱

Reply:
...I found the scary code, and it is right here in Backbase Streams [UserProfileMapper.mapName](https://github.com/Backbase/stream-services/blob/ecb050797a78f71c5472915c732249f3d6b4b88c/stream-access-control/access-control-core/src/main/java/com/backbase/stream/mapper/UserProfileMapper.java#L43-L56)

Reply:
these are automatically recorded request/responses by wiremock and turned into stub mappings used to serve the mocked requests. `fullName` is part of the user-manager api specs.

Reply:
I'll try to enhance this stream mapper to set it up accordingly

Reply:
After doing a quick check I can see on the spec that the user-manager has major inconsistencies in the api specs, a few apis have only full name, other support fine graining it to family and given names. I'll have to leave it be. e.g.

https://backbase.io/developers/apis/specs/user-manager/user-manager-service-api/2.11.2/models/IdentityListedItem/

https://backbase.io/developers/apis/specs/user-manager/user-manager-service-api/2.11.2/models/CreateIdentityRequest/

Reply:
I mentioned these comments to the Identity team... the least we can do.
There is nothing that can be done here & now.